cmd/serverless/main.go

```diff
@@ -252,6 +257,21 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 
 	wg.Wait()
 
+	coldStartSpanCreator := &trace.ColdStartSpanCreator{
+		CreateSpan:            &sync.Once{},
+		SyncSpanDurationMutex: &sync.Mutex{},
```

Comment:
Embedding the mutex (see comment below) will allow you to remove this line and thus increase (decrease?) your all important code golf score.